### PR TITLE
feat: added cart and wishlist page

### PR DESCRIPTION
**Pull request by bishnoimukesh**
# Change Logs

## Live preview link: 
https://brand-factory-react.netlify.app/

### CODE REVIEW

File: src/pages/WishListPage/WishListPage.jsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    const {wishState:{wishList}} = useCartContext();
+    return (
+        <>
+            <Nav />
+            <div className="wishlist-container">
+                <div className="wishlist-heading">
+                    <h1>Wishlist</h1>
+                </div>
+                <div className="wishlist-items-container">
+                    {/* <p>Your wishlist is Empty</p> */}
```
Comment (pnchinmay) [inline src/pages/WishListPage/WishListPage.jsx:17]: Remove these comments

**Review by pnchinmay — APPROVED** [review]
Left comments
Everything else looks good :100:

File: src/pages/WishListPage/WishListPage.jsx (hunk 1 above)
Comment (bishnoimukesh) [reply src/pages/WishListPage/WishListPage.jsx:17]: ok i will

File: src/pages/CartPage/CartPage.jsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import './CartPage.css';
+import {Nav} from './../../components/Nav/Nav';
+import { VerticalCard } from '../../components/Card/VerticalCard';
+import {useCartContext} from './../../Context/CardContext';
```
Comment (devdeadviz) [inline src/pages/CartPage/CartPage.jsx:5]: Try to name your folder names in lowercase.

File: src/pages/CartPage/CartPage.jsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+import {PriceCard} from './../../components/Card/PriceCard';
+
+const CartPage = () => {
+    const {cartState:{cart}} = useCartContext();
+    // console.log(cart);
```
Comment (devdeadviz) [inline src/pages/CartPage/CartPage.jsx:10]: Remove console statements.

File: src/pages/CartPage/CartPage.jsx [hunk 4]
```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import './CartPage.css';
+import {Nav} from './../../components/Nav/Nav';
+import { VerticalCard } from '../../components/Card/VerticalCard';
```
Comment (devdeadviz) [inline src/pages/CartPage/CartPage.jsx:4]: You can use an `index.js` file in the components folder and add all your components exports in that. By this you can import all your components from a single file.

**Review by devdeadviz — APPROVED** [review]
The code is clean and your file structure is good. Good Work!

File: src/pages/CartPage/CartPage.jsx (hunk 3 above)
Comment (bishnoimukesh) [reply src/pages/CartPage/CartPage.jsx:10]: noted.

File: src/pages/CartPage/CartPage.jsx (hunk 4 above)
Comment (bishnoimukesh) [reply src/pages/CartPage/CartPage.jsx:4]: i will implement that.

**Pull request merged**